app-benchmarks/stress-ng/files/makefile.patch

```diff
@@ -0,0 +1,75 @@
+--- a/Makefile	2018-04-27 17:02:16.000000000 +0200
++++ b/Makefile	2018-05-05 00:15:00.687399505 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -DVERSION='"$(VERSION)"'
+ 
+ #
+ # Pedantic flags
+@@ -314,12 +314,10 @@
+ .o: stress-ng.h Makefile
+ 
+ .c.o: stress-ng.h Makefile $(SRC)
+-	@echo "CC $<"
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) $(LDFLAGS) -c -o $@ $<
+ 
+ stress-ng: info $(OBJS)
+-	@echo "LD $@"
+-	@$(CC) $(CPPFLAGS) $(CFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
++	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) $(OBJS) -lm $(LDFLAGS) -lc -o $@
+ 	@sync
+ 
+ .PHONY: info
+@@ -340,7 +338,7 @@
+ 	@echo "};" >> apparmor-data.c
+ 	@echo "const size_t g_apparmor_data_len = sizeof(g_apparmor_data);" >> apparmor-data.c
+ 	@echo "CC $<"
+-	@$(CC) -c apparmor-data.c -o apparmor-data.o
++	$(CC) $(CFLAGS) $(LDFLAGS) -c apparmor-data.c -o apparmor-data.o
+ 	@rm -rf apparmor-data.c
+ 
+ #
+@@ -353,13 +351,11 @@
+ stress-personality.c: personality.h
+ 
+ perf.o: perf.c perf-event.c
+-	@$(CC) -E perf-event.c | grep "PERF_COUNT" | sed 's/,/ /' | awk {'print "#define _SNG_" $$1 " (1)"'} > perf-event.h
+-	@echo CC $<
+-	@$(CC) $(CFLAGS) -c -o $@ $<
++	$(CC) $(CFLAGS) $(LDFLAGS) -E perf-event.c | grep "PERF_COUNT" | sed 's/,/ /' | awk {'print "#define _SNG_" $$1 " (1)"'} > perf-event.h
++	$(CC) $(CFLAGS) $(LDFLAGS) -c -o $@ $<
+ 
+ stress-vecmath.o: stress-vecmath.c
+-	@echo CC $<
+-	@$(CC) $(CFLAGS) -fno-builtin -c -o $@ $<
++	$(CC) $(CFLAGS) $(LDFLAGS) -fno-builtin -c -o $@ $<
+ 	@touch stress-ng.c
+ 
+ $(OBJS): stress-ng.h Makefile
+@@ -398,10 +394,8 @@
+ 	STRESS_NG=./stress-ng debian/tests/fast-test-all
+ 
+ .PHONY: install
+-install: stress-ng stress-ng.1.gz
++install: stress-ng
+ 	mkdir -p ${DESTDIR}${BINDIR}
+ 	cp stress-ng ${DESTDIR}${BINDIR}
+-	mkdir -p ${DESTDIR}${MANDIR}
+-	cp stress-ng.1.gz ${DESTDIR}${MANDIR}
```

Comment:
Why not fix the rule?

Reply:
You mean uncompress man, and remove here the *.gz?